### ISSUE TITLE
Adding additional circles on an existing line chart

### ISSUE THREAD

**Issue by sobbuh** [danielgindi/Charts#4466]
I currently have a chart as seen in the image [here](https://i.stack.imgur.com/jjwfp.png) -- it's a simple line chart with a dataset of [x: minutes since start of day, y: Int (from 1-10)]. 

I have additional "notes" with [x: minutes since start of day, y: String]. 

Using those X coordinates, I want to calculate the y value on the existing line and then add a point on the chart at that x,y coordinate with a different color circle and some way to access the relevant string (i.e. I want to add the grey dot as seen [here](https://i.stack.imgur.com/kkjyr.png).  When the user clicks on the grey dot, I want to be able to access the string to show it in another view)

Is this possible?

**Comment by xsleonard** [comment]
Yes, add another dataSet for the notes dots and configure it to draw a grey dot. There are delegate methods for handling tapped points (called "highlight"), and you can attach metadata to each data point (see `ChartEntry.data`). If you can't disable the line for the notes dataSet you could try configuring it with .clear color.
